### PR TITLE
Add custom models to import/export functionality of CMS

### PR DESCRIPTION
**Pull request by defaye**
Will need testing

### CODE REVIEW

**Comment by Levia** [comment]
I'm now getting a file/directory not found error at the following path when running the exporter:

```
path/to/ProtectedPlanet/db/cms_seeds/protectedplanet/home_carousel_slide.json
```

I run the exporter like the following:
```
rake 'comfy:cms_seeds:export[protectedplanet, protected-planet]'
```

I believe that the path it should be looking at is `protected-planet/home_carousel_slide.json` instead of `protectedplanet/home_carousel_slide.json`.

Am I doing something wrong?

**Comment by Levia** [comment]
@stacytalbot I have the same with CTAs if I make the exporter ignore the carousel

File: .env.example [hunk 1]
```diff
@@ -31,3 +31,4 @@ API_COUNTRY_NAME_ATTRIBUTE=XXXXXXXXXX
 API_JRC_COUNTRY_AREA_ATTRIBUTE=XXXXXXXXXX
 API_JRC_TERR_AREA_ATTRIBUTE=XXXXXXXXXX
 
+COMFY_CMS_INCLUDED_EXPORT_MODELS="HomeCarouselSlide CallToAction"
```
[the remark this answers is not in the document]
Comment (defaye) [reply .env.example:34]: I see what you're saying -- can make this change!

**Comment by Levia** [comment]
Thanks @stacytalbot , it seems my CMS was slightly misconfigured then.
I run both commands and seems to work fine :)

So it's just about those minor comments I left now

**Comment by defaye** [comment]
Does this mean the Carousel data should not be included in the export/import?

**Comment by defaye** [comment]
@stacytalbot  @Levia changes requested by Ferdi have now been implemented~

**Pull request merged**